Private link\endpoints update (#3888)
astronomer/docs: astro/connect-aws.md

- Elastic Load Balancing (ELB)  - Interface Endpoint
- AWS Security Token Service (AWS STS) - Interface Endpoint

> To request additional endpoints, or assistance connecting to other AWS services please follow the next steps:

Reviewer: ```suggestion
To request additional endpoints, or assistance connecting to other AWS services, complete the following steps:
```

#### AWS Service Endpoint

> - Prepare a list of the AWS Services which require Endpoints (e.g. SQS, Lambda, DynamoDB).

Reviewer: ```suggestion
- Prepare a list of your AWS Services that require Endpoints, such as SQS, Lambda, or DynamoDB.
```

Author: Can we add "including, but not limited to SQS, Lambda, or DynamoDB"?

> - Contact [Astronomer support](https://cloud.astronomer.io/open-support-request) and provide this information for the further steps.

Reviewer: ```suggestion
- Contact [Astronomer support](https://cloud.astronomer.io/open-support-request) and provide this information for next steps.
```

By default, Astronomer support activates the **Enable DNS Name** option on supported AWS PrivateLink endpoint services.  With this option enabled, you can make requests to the default public DNS service name instead of the public DNS name that is automatically generated by the VPC endpoint service. For example, `*.notebook.us-east-1.sagemaker.aws` instead of `vpce-xxx.notebook.us-east-1.vpce.sagemaker.aws`. For more information about AWS DNS hostnames, see [DNS hostnames](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-dns.html#:~:text=recursive%20DNS%20queries.-,DNS%20hostnames,-When%20you%20launch).

#### Custom VPC Endpoint

> - Open a list of your clusters in [Cloud UI](https://cloud.astronomer.io/clusters).

Reviewer: ```suggestion
- Open a list of your clusters in the [Astro UI](https://cloud.astronomer.io/clusters).
```

> - Click on the target cluster and get a value for **ACCOUNT ID** field.
> - Go to your VPCe Service configuration and add `arn:aws:iam::<Astro account_id>:role/astronomer-remote-management` into a list of the Allowed principals.
> - Contact [Astronomer support](https://cloud.astronomer.io/open-support-request) and provide a VPCe Service name for the custom Service.
> - Add to request a preferable DNS name for Endpoint, if Route53 alias is required for the proper connectivity (i.e. Snowflake, EKS). Otherwise you can skip this step.

Reviewer: ```suggestion
- Click on the target cluster and retrieve the value for the `ACCOUNT ID` field.
- Go to your VPCe Service configuration and add `arn:aws:iam::<Astro account_id>:role/astronomer-remote-management` into a list of the Allowed principals.
- Contact [Astronomer support](https://cloud.astronomer.io/open-support-request) and provide a VPCe Service name for the custom Service.
- (Optional)  If Route53 alias is required for the proper connectivity by your service such as Snowflake EKS, add a preferable DNS name for the Endpoint to your request. Otherwise, you can skip this step.
```

Author: "such as Snowflake or EKS"
